### PR TITLE
Bl add icons

### PR DESCRIPTION
**Pull request by LINDGRENBA**
## Description
This PR will add icons to help make the user experience more enjoyable, create a clean / fun look and feel and simplify the design. This PR will also make some updates to the styling of the navigation to update the UI. 

## Related Issue
closes [#42](https://github.com/the-collab-lab/tcl-17-smart-shopping-list/issues/42)

## Acceptance Criteria

- [ ] Icons should be understood universally
- [ ] The purpose and/or intent of the icons should be clear
- [ ] Use icons minimally to keep the look of the app clean
- [ ] Icons should not make the application harder to use
- [ ] Add font awesome shopping cart, trash can, list icons
- [ ] Shopping bag from flaticon

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles: New feature     |
|  ✓ | :hammer: Refactoring       |

### Before
before-icons

### After
after-icons

## Testing Steps / QA Criteria
* One way to test the site is to click on the `details` button below to open the `Netlify preview`

Screen Shot 2021-01-12 at 6 05 51 PM

* Otherwise from your terminal, pull down this branch with `git pull origin BL-add-icons` and check that branch out with `git checkout BL-add-icons`
* `npm start` to launch the app.
* confirm that you see the following icons
1.  shopping cart icon with plus sign for the `Add Item` nav link
2.  list icon for the `Shopping List` nav link
3.  shopping bag svg for the `Home` nav link
4. trash can icon for each list item

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -26,6 +26,8 @@
     -->
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Fira+Sans:wght@300;400;500;700&display=swap" />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/js/all.js" integrity="sha256-2JRzNxMJiS0aHOJjG+liqsEOuBb6++9cY4dSOyiijX4=" crossorigin="anonymous"></script>
     <title>React App</title>
```
Comment (LINDGRENBA) [inline public/index.html:31]: Github is still marking this as a conflict, so I can just be sure to resolve the conflict prior to merging. This is the correct version here with both the google font link and the font awesome script.

File: src/App.css [hunk 2]
```diff
@@ -18,19 +18,19 @@ nav a {
...
   color: black;
-  font-size: 1.1rem;
-  padding: 3% 0;
+  font-size: 1rem;
+  padding: 2% 0;
   margin: 2%;
   width: 30%;
-  border: 3px solid black;
+  border-radius: 5px;
 }
```
Comment (LINDGRENBA) [inline src/App.css:30]: I altered the css here just a little to get the style a little closer to our wireframe. I am realizing I should move this styling into the Navigation.css though, so will do that and push those changes to this branch.

File: src/components/List.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ import CircleUnchecked from '@material-ui/icons/RadioButtonUnchecked';
 import CircleCheckedFilled from '@material-ui/icons/CheckCircle';
 
 const List = ({ token }) => {
-  const { docs, errorMessage, deleteDoc } = useFirestore(token);
+  const { docs, errorMessage, deleteDoc, loading } = useFirestore(token);
```
Comment (LINDGRENBA) [inline src/components/List.js:13]: This was a change that Rachel and I made earlier this week.

File: src/components/Navigation.js [hunk 4]
```diff
@@ -1,17 +1,29 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
+import homeIcon from './../img/groceries.svg';
+import './../styles/Navigation.css';
 
-const Navigation = () => {
+const Navigation = ({ confirmDeleteUserToken }) => {
```
Comment (LINDGRENBA) [inline src/components/Navigation.js:6]: This incorporates changes made by @rachel-fischoff to the Navigation, but integrates them with my changes to include the SVG.

File: src/styles/Navigation.css [hunk 5]
```diff
@@ -0,0 +1,36 @@
+nav {
```
Comment (LINDGRENBA) [inline src/styles/Navigation.css:1]: These styles moved from App.css to Navigation.css

**Review by DeeDowns — APPROVED** [review]
This looks good to me. 
My only comment, to have consistency what should be the order of the icons their labels? 
For example, in the navigation at the bottom of the `List` view we have the icon then the label. (eg the shopping bag icon then the `Home` label). 
Screen Shot 2021-02-25 at 2 28 08 PM

Then in the `Add Item` view for the button has the label then the icon. 
Screen Shot 2021-02-25 at 2 28 20 PM

In the feature I worked on (adding the `Add Item` button to the top of the `List` view) my button has the icon then the label. 
Screen Shot 2021-02-25 at 2 28 34 PM

Just wondering if we should decide an order, for a uniform look.

**Review by rachelrohrbach — APPROVED** [review]
It looks good Brittany! I really like how the bottom navigation links look with the icons

**Comment by rachelrohrbach** [comment]
> This looks good to me.
> My only comment, to have consistency what should be the order of the icons their labels?
> For example, in the navigation at the bottom of the `List` view we have the icon then the label. (eg the shopping bag icon then the `Home` label).
> Screen Shot 2021-02-25 at 2 28 08 PM
> 
> Then in the `Add Item` view for the button has the label then the icon.
> Screen Shot 2021-02-25 at 2 28 20 PM
> 
> In the feature I worked on (adding the `Add Item` button to the top of the `List` view) my button has the icon then the label.
> Screen Shot 2021-02-25 at 2 28 34 PM
> 
> Just wondering if we should decide an order, for a uniform look.

@DeeDowns I agree we should decide on an order. I like the icon and then the text/label order.

**Comment by LINDGRENBA** [comment]
I agree about having the order be consistent. I don't have any strong feelings, so I'll defer to the rest of the group on that one 😄

**Pull request merged**